### PR TITLE
[DatePicker | TimePicker] Update visual styles

### PR DESCRIPTION
**Pull request by almedina-ms**
Updates the visual style for DatePicker and TimePicker elements

## Description
* Changes background  color of dropdown to the new standard color for flyouts.
* Selected items (rows) now have a rounded, accent color backplate. This backplate has a 4px margin on the left/right and 2px margin on the top/bottom.
* Hovered items have a lighter gray rounded backplate, with the same margins as the selected row.
* The scroll up and down visuals use the new, updated icons. When the scroller arrow is hovered, it has a rounded grey backplate with ample margins from the edge of the control
* The check and X buttons at the bottom of the Date/Time Picker use the new iconography. On hover, these buttons receive a 4px rounded gray backplate with ample margins from the edge of the control. 
* For the closed Date/Time Picker focus, it is now filled with an accent color background

## Motivation and Context
Visual updates

## How Has This Been Tested?
The changes must be approved with design discussions

### DatePicker 

| | | | |
| --- | --- | --- | --- |
| image | image | image | image |
| image | image | image | image |
| image | image | image | image |

### TimePicker
| | | | |
| --- | --- | --- | --- |
| image | image | image | image | 
| image | image | image | image | 
| image | image | image | image |

### CODE REVIEW

**Comment by ranjeshj** [comment]
@almedina-ms I think these files moved into common styles. Can you merge in master and update the files ?

File: dev/DatePicker/DatePicker_themeresources.xaml [hunk 1]
```diff
@@ -4,6 +4,7 @@
   xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
   xmlns:contract7NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,7)"
   xmlns:contract6Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,6)"
+  xmlns:primitives="using:Microsoft.UI.Xaml.Controls.Primitives"
```
Comment (ranjeshj) [inline dev/DatePicker/DatePicker_themeresources.xaml:7]: If there isnt one already, can you add a page to the common styles page (or another page) and add a test to load that page and verify that the control loads and you able to invoke the flyout ?

File: dev/CommonStyles/DatePicker_themeresources.xaml [hunk 2]
```diff
@@ -226,6 +233,7 @@
                                                     BorderThickness="{ThemeResource DatePickerBorderThemeThickness}"
                                                     Content="{TemplateBinding Content}"
                                                     Foreground="{TemplateBinding Foreground}"
+
```
Comment (StephenLPeters) [inline dev/CommonStyles/DatePicker_themeresources.xaml:236]: Nit: remove added space

File: dev/CommonStyles/DateTimePickerFlyout_themeresources.xaml [hunk 3]
```diff
@@ -97,11 +131,173 @@
...
-                            contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}"/>
+                            contract7NotPresent:CornerRadius="{TemplateBinding CornerRadius}"/>
                     </Grid>
                 </ControlTemplate>
             </Setter.Value>
         </Setter>
     </Style>
 
+    <!-- Default style for Windows.UI.Xaml.Controls.Primitives.LoopingSelector -->
+    <Style TargetType="LoopingSelector" BasedOn="{StaticResource DefaultLoopingSelectorStyle}" />
```
[the remark this answers is not in the document]
Comment (almedina-ms) [reply dev/CommonStyles/DateTimePickerFlyout_themeresources.xaml:143]: I couldn't find usages for looping selector in any control, I believe time picker and date picker set this control from code so I'd understand that only these control use the Looping selector

File: dev/DatePicker/DatePicker_themeresources.xaml (hunk 1 above)
Comment (almedina-ms) [reply dev/DatePicker/DatePicker_themeresources.xaml]: CommonTestStyles have a test for DatePicker and TimePicker and both of them click on their respective buttons to invoke the flyout

**Comment by chingucoding** [comment]
@almedina-ms I think you had a bad merge here, there shouldn't be so many commits.

File: dev/CommonStyles/Common_themeresources_any.xaml [hunk 4]
```diff
@@ -73,6 +73,7 @@
             <Color x:Key="SurfaceStrokeColorInverse">#0F000000</Color>
 
             <Color x:Key="DividerStrokeColorDefault">#15FFFFFF</Color>
+            <Color x:Key="DividerStrokeColorFocus">#33000000</Color>
```
[the remark this answers is not in the document]
Comment (mdtauk) [reply dev/CommonStyles/Common_themeresources_any.xaml:76]: Shouldn't it be ``#33FFFFFF`` also, as a brighter version of the Default Divider Stroke?

[the remark this answers is not in the document]
Comment (almedina-ms) [reply dev/CommonStyles/Common_themeresources_any.xaml]: the added colors were part of a removed feature where we would style the focused element in a different way, as it's no longer done then those added colors/brushes are unnecessary and as such, removed

File: dev/CommonStyles/TimePicker_themeresources.xaml [hunk 5]
```diff
@@ -16,23 +16,30 @@
...
+            <StaticResource x:Key="TimePickerButtonBackgroundFocused" ResourceKey="AccentAAFillColorDefaultBrush" />
+
             <StaticResource x:Key="TimePickerButtonForeground" ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="TimePickerButtonForegroundDefault" ResourceKey="TextFillColorSecondaryBrush" />
             <StaticResource x:Key="TimePickerButtonForegroundPointerOver" ResourceKey="TextFillColorPrimaryBrush" />
-            <StaticResource x:Key="TimePickerButtonForegroundPressed" ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="TimePickerButtonForegroundPressed" ResourceKey="TextFillColorSecondaryBrush" />
             <StaticResource x:Key="TimePickerButtonForegroundDisabled" ResourceKey="TextFillColorDisabledBrush" />
-            <StaticResource x:Key="TimePickerButtonForegroundFocused" ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="TimePickerButtonForegroundFocused" ResourceKey="TextOnAccentAAFillColorPrimary" />
```
[the remark this answers is not in the document]
Comment (almedina-ms) [reply dev/CommonStyles/TimePicker_themeresources.xaml]: This resource is not used so it was removed

**Review by marksfoster — APPROVED** [review]

**Pull request merged**